stardist/bioimageio_utils.py

```diff
@@ -453,4 +453,13 @@ def import_bioimageio(source, outpath):
 
     model_class = (StarDist2D if config['n_dim'] == 2 else StarDist3D)
     model = model_class(None, outpath.name, basedir=str(outpath.parent))
+
+    # update the location of the stardist specific files, since they are the ones being used by the model
+    biomodel.config['stardist']['config'] = outpath / 'config.json'
+    biomodel.config['stardist']['thresholds'] = outpath / 'thresholds.json'
+    biomodel.config['stardist']['weights'] = outpath / weights
+    # add the resource description as an attribute to the model,
+    # so that additional information stored in there does not get lost
+    model.bioimageio_resource_description = biomodel
```

Comment:
But this is a fleeting property that is not persisted anywhere.

Reply:
I am not quite sure what you mean by "is not persisted anywhere", but this attribute is accessible on the object that is returned:
```python
from stardist import import_bioimageio
model = import_bioimageio("10.5281/zenodo.6348084", "./bio-model")
print(hasattr(model, "bioimageio_resource_description"))
# prints True
```

Reply:
What I mean is that `import_bioimageio` should only be called once to create a copy of the model. After that, the model can/should be loaded from disk as any regular StarDist model. And the `bioimageio_resource_description` property will be lost in this case, because it is never written to any persistent storage.

```python
from stardist.models import StarDist2D
model = StarDist2D(None, 'bio-model', basedir='.')
print(hasattr(model, "bioimageio_resource_description"))
# prints False
```

Reply:
Ok, I see what you mean. See comment below.